K2Bridge/Visitors/MetricAggregationVisitor.cs

```diff
@@ -31,6 +31,15 @@ public void Visit(CardinalityAggregation cardinalityAggregation)
             cardinalityAggregation.KustoQL = $"['{cardinalityAggregation.FieldAlias}']={KustoQLOperators.DCount}({cardinalityAggregation.FieldName})";
         }
 
+        /// <inheritdoc/>
+        public void Visit(MinAggregation minAggregation)
```

Comment:
Nit: in all places except here max comes before min. Personally I think min should become before max, but it's up to you, just stay consistent.

Reply:
Indeed, make more sense. More readable, from a human perspective